prefix_subarray.py

```
@@ -0,0 +1,27 @@
+from collections import defaultdict
+
+def subarray_sum(nums, k):
```

Ruff (0.14.1)

Unused function argument: `k`

(ARG001)